app/src/main/java/com/example/internetshop/presentation/activity/fragments/ProductDetailsFragment.kt

```diff
@@ -55,6 +55,18 @@ class ProductDetailsFragment : BaseFragment() {
         component.inject(this)
     }
 
+    override fun getTitle(): String {
```

Comment:
The same. All three methods should be in body style

Reply:
fixed